contracts/sdjwt-verifier/src/state.rs

```diff
@@ -0,0 +1,19 @@
+// State structure
+
+use std::collections::HashMap;
+
+use avida_common::types::{IssuerSourceOrData, RouteId};
+use cosmwasm_std::Addr;
+use cw_storage_plus::{Item, Map};
+
+use crate::types::{PendingRoute, VerificationRequirements};
+
+pub const MAX_PRESENTATION_LENGTH: Item<usize> = Item::new("max_presentation_length");
```

Comment:
I would say some of these states can be moved to the package to be shared across verifier implementations.

For example, we have the type`VerfiablePresentation` as `Binary` in the package because it will be required for any verifier, sd-jwt, anoncreds, zk, etc.

Reply:
Moved `MAX_PRESENTATION_LENGTH` to package, let's see what we can reuse after we develop other verifiers.
https://github.com/nymlab/AVIDA/pull/19/commits/52c864a9ccac2de41aebf1787b66cf7008502adc